src/app/profile-settings/profile-settings.page.ts

```diff
@@ -639,13 +640,15 @@ export class ProfileSettingsPage implements OnInit, OnDestroy, AfterViewInit {
           PageId.ONBOARDING_PROFILE_PREFERENCES, profile, 'manual', Environment.ONBOARDING
         );
         const correlationlist: Array<CorrelationData> = [];
```

Comment:
correlationlist.push({id: CorReleationDataId.MANUAL_PROFILE, type: CorReleationDataType.MANUAL_PROFILE});
Here type should be PageId